x-pack/test/functional_search/tests/search_index_details.ts

```diff
@@ -62,7 +62,6 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
           await esDeleteAllIndices(indexName);
         });
         it('can load index detail page', async () => {
-          await pageObjects.searchNavigation.navigateToIndexDetailPage(indexName);
```

Comment:
why are we removing this?

Ideally we should be testing the index details page is the expected page for both classic and solution nav. classic would be the index management details page. solution nav would be the search indices details page.

Reply:
We have same navigation in before section